eclass/mate-desktop.org.eclass

```diff
@@ -50,15 +50,11 @@ inherit versionator
 
 # Set SRC_URI or EGIT_REPO_URI based on whether live
 if [[ ${PV} == 9999 ]]; then
-	EGIT_REPO_URI="
-		https://github.com/mate-desktop/${MATE_DESKTOP_ORG_PN}.git
-		git://github.com/mate-desktop/${MATE_DESKTOP_ORG_PN}.git
-		http://github.com/mate-desktop/${MATE_DESKTOP_ORG_PN}.git
-	"
+	EGIT_REPO_URI="https://github.com/mate-desktop/${MATE_DESKTOP_ORG_PN}.git"
```

Comment:
I spoke with github support.  They said they aren't planning on supporting SSHFP records at this time, so I guess I'm okay with removing git:// for the time being if we are doing that everywhere else.

@mgorny wrt the git-r3 eclass, what do you think about a config option allowing passing the ssh option "VerifyHostKeyDNS yes"?